trogon/trogon.py

```diff
@@ -259,14 +275,18 @@ def run(
                     )
 
                     split_app_name = shlex.split(self.app_name)
-                    program_name = shlex.split(self.app_name)[0]
+                    program_name = split_app_name[0]
                     arguments = [*split_app_name, *self.post_run_command]
-                    os.execvp(program_name, arguments)
+                    # update PATH to include current working dir.
+                    env: dict[str, str] = os.environ.copy()
+                    env["PATH"] = os.pathsep.join([os.getcwd(), env["PATH"]])
+                    os.execvpe(program_name, arguments, env)
 
     @on(CommandForm.Changed)
     def update_command_to_run(self, event: CommandForm.Changed):
-        include_root_command = not self.is_grouped_cli
-        self.post_run_command = event.command_data.to_cli_args(include_root_command)
+        self.post_run_command = event.command_data.to_cli_args(
+            include_root_command=False
+        )
```

Comment:
This was causing an issue for me, and I could not think of a single use-case that would need to include the root-command name; it inherently doesn't have a name. :shrug: 

With this consistently set to `False`, it proved to work fine for all examples, existing and new.